Automattic/jetpack
# jetpack-live-branches: Update to support multiple plugins

#### Changes proposed in this Pull Request:

The Jetpack Beta Tester plugin v3.0.0 will support multiple plugins.

https://github.com/Automattic/jurassic.ninja/pull/239 updates Jurassic
Ninja to work with that.

This updates the jetpack-live-branches TamperMonkey script to work with
the Jurassic Ninja update.

This will work even before the JN PR is merged and deployed, but we might want to wait at least until we've confirmed that the new API is what we want.

#### Jetpack product discussion


p9dueE-362-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install the new version of the script. See if it works.
* Adjust the `const host` on line 16 to point to a JN installation running with the updated PR. See that it offers all the available plugins. See if it works too, assuming that we've done the Beta 3.0.0 release (or you've hacked your copy of JN to use Beta master instead of the latest release).
